src/web/make-report.rkt

```diff
@@ -99,14 +102,21 @@
               (div ([id "subreports"] [style "display: none"]))))))
 
       (div ((id "large"))
-       ;; TODO minutes
        ,(render-large "Average Accuracy"
-                      ;; TODO ulps->bits ?
-                      (format-accuracy total-start total-bits #:unit "%")
+                      (format-accuracy total-start maximum-accuracy #:unit "%")
                       " → "
-                      (format-accuracy total-result total-bits #:unit "%"))
+                      (format-accuracy total-result maximum-accuracy #:unit "%"))
        ,(render-large "Time" (format-time total-time #:max 'minute))
-       ,(render-large "Crashes and Timeouts" (~a (+ total-crashes total-timeouts)) "/" (~a total-tests)))
+       ,(render-large "Crashes and Timeouts" (~a (+ total-crashes total-timeouts)) "/" (~a total-tests))
+       ,(render-large "Speedup at Initial Accuracy"
+                      (match-let ([(list cost _)
+                                   (argmin
+                                    (match-lambda [(list _ accuracy)
+                                                   (-
+                                                    (- 1 (/ accuracy maximum-accuracy))
+                                                    (- 1 (/ initial-accuracy maximum-accuracy)))])
```

Comment:
Two comments here. First, I don't understand this code. Isn't this just getting the lowest accuracy? I don't think we want that. I think we want something like `(first (filter (match-lambda [(list _ accuracy) (> accuracy initial-accuracy)]) frontier))`

And second, let's move it to a definition before the actual HTML generation starts, just for readability.